CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Bugfix: Fixed hidden tooltips when always on top is active (#2384)
```

Comment:
This changelog entry should be lower, next to other entries prefixed with `Bugfix`

Reply:
Done